gmm_client/test/basic_SUITE.erl

```diff
@@ -0,0 +1,102 @@
+-module(basic_SUITE).
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+-export([
+    all/0,
+    groups/0,
+    init_per_suite/1,
+    end_per_suite/1,
+    init_per_group/2,
+    end_per_group/2
+]).
+-export([
+    vertices_test/1,
+    edges_test/1
+]).
+
+groups() -> [{basic_flow, [sequence], [
+    vertices_test,
+    edges_test
+]}].
+
+all() -> [{group, basic_flow}].
+
+init_per_suite(Config) ->
+    [{ct_hooks, [docker_compose_cth]} | Config].
+
+end_per_suite(_Config) ->
+    ok = application:stop(gmm_client).
+
+init_per_group(basic_flow, _Config) ->
+    timer:sleep(1000), %% from unknown reasons some delay is needed before executing first tests - without it tests fail with reason {badmatch, {error, closed}}
+    {ok, _} = application:ensure_all_started(gmm_client),
+    ok;
+init_per_group(_Group, _Config) ->
+    ok.
+
+end_per_group(_Group, _Config) ->
+    ok.
+
+vertices_test(_Config)->
+    % given
+    client:add_user("Jan"),
+    client:add_group("Friends"),
+    client:add_space("Space"),
+    client:add_provider("Provider"),
+    % when
+    UserId = client_test_helper:get_id_from_the_list(),
+
+    % then
+    [
+      ?assertEqual("Jan", client_test_helper:get_vertex_name(UserId))
+    ],
+    % when
+    client:delete_vertex(UserId),
+    SpaceId = client_test_helper:get_id_from_the_list(),
+
+    % then
+    [
+      ?assertEqual("Space", client_test_helper:get_vertex_name(SpaceId))
+    ],
```

Comment:
Proponuję tu sprawdzić, czy usuniętego wierzchołka faktycznie nie ma itd.
Najlepiej po prostu mieć listę wierzchołków {typ, nazwa}, które próbowaliśmy utworzyć, po czym przy każdym dodawaniu/usuwaniu robić to samo "lokalnie" na liście, i porównywać wyniki listingu

Reply:
gotowe